deploy.sh

```diff
@@ -14,8 +15,10 @@ npm install
 npm run build
 
 sentry-cli releases -o $SENTRY_ORG new -p $SENTRY_PROJECT $RELEASE
+sentry-cli releases -o $SENTRY_ORG finalize -p $SENTRY_PROJECT $RELEASE
```

Comment:
does it matter if you call finalize after `set-commits` or after?

Reply:
@ndmanvar any thoughts here? Can we get your review on this PR?